evalml/tests/automl_tests/parallel_tests/test_automl_dask.py

```diff
@@ -36,6 +36,7 @@ def sequential_results(X_y_binary_cls):
     return seq_results
 
 
+@pytest.mark.xfail
```

Comment:
do we have an issue tracking to remove this once we can support it? 😄

Reply:
We do now.